server/app/services/export/PdfExporter.java

```diff
@@ -70,15 +74,23 @@ public InMemoryPdf export(Application application) throws DocumentException, IOE
             answers,
             applicantNameWithApplicationId,
             application.getProgram().getProgramDefinition(),
-            application.getLatestStatus());
+            application.getLatestStatus(),
+            getSubmitTime(application.getSubmitTime()));
     return new InMemoryPdf(bytes, filename);
   }
 
+  private Optional<String> getSubmitTime(Instant submitTime) {
+    return submitTime == null
+        ? Optional.empty()
+        : Optional.of(dateConverter.renderDateTime(submitTime));
```

Comment:
```suggestion
    return submitTime == null
        ? "Not yet submitted"
        : string.format("Submit Time: %s", dateConverter.renderDateTime(submitTime));
```

Reply:
I used the default text here.

Reply:
Application submitted without submission time marked.